addon/ng2/commands/set.ts

```diff
@@ -1,25 +1,25 @@
 import * as Command from 'ember-cli/lib/models/command';
-import {CliConfig} from '../models/config';
+import {Config} from '../models/config';
 
 
 const SetCommand = Command.extend({
   name: 'set',
   description: 'Set a value in the configuration.',
-  works: 'outsideProject',
+  works: 'everywhere',
 
   availableOptions: [
     { name: 'global', type: Boolean, default: false, aliases: ['g'] },
   ],
 
   run: function (commandOptions, rawArgs): Promise<void> {
     return new Promise(resolve => {
-      const config = new CliConfig();
-      config.set(rawArgs[0], rawArgs[1], commandOptions.force);
+      let config = new Config();
+      config.validatePath(rawArgs[0]);
+      config.set(config.config, rawArgs[0], rawArgs[1]);
```

Comment:
Could you do a `a=b` kinda of syntax?

Basically:
```typescript
for (let arg of rawArgs) {
  let [key, value] = arg.split('=', 1);
  config.validatePath(key);
  config.set(key, value);
}```

What do you think?  This way we can `ng config set key[1]=value1 key[2]=value2`.